### DISCUSSION TITLE
Some confuse about The @output decorator can only use once

### DISCUSSION THREAD

**Discussion by imoyao** [apiflask/apiflask#113] [Q&A]
From the document [here](https://apiflask.com/usage/#use-output-to-format-response-data) i know that "The @output decorator can only use once",but how can i deal with this condition?
```python
# the code is not working
@bp.get('/')
@input(PaginationSchema, 'query')
@input(EmptySchema, 'query')
@output(FundPaginationSchema) 
@output(FundSampleSchema(many=True))   # here is my trouble
def funds(query, data):
    """
    获取基金信息
    :param query:
    :return:
    """
    if not data:
        ret= Fund.query.order_by(Fund.fund_code.desc()).all()   # get all funds
    else:
        # pagination
        pagination = paginate_query(Fund, query)
        _items = pagination.items
        ret = {'funds': _items, 'pagination': pagination_builder(pagination)}
    return ret
```
In my assume，the input of `PaginationSchema` can be output by `FundPaginationSchema` while `EmptySchema` can be deal with `FundSampleSchema`.
In DRF([Django REST framework](https://www.django-rest-framework.org/)),it can be deal with code like this:
```python
# the code is not working
page = self.paginate_queryset(queryset)
if page is not None:
    serializer = self.get_serializer(page, many=True)
    return self.get_paginated_response(serializer.data)

serializer = self.get_serializer(queryset, many=True)
return Response({'status': 200, 'msg': 'success', 'data': {'data': serializer.data}})

```
In order to avoid my question not clear enough, here are two supplementary screenshots of my question：
- not request args
![Snipaste_2021-07-28_17-59-23](https://user-images.githubusercontent.com/22978431/127304282-f3d70a40-be18-43ac-84c2-9e87c514dd96.png)
- with args
![Snipaste_2021-07-28_18-01-43](https://user-images.githubusercontent.com/22978431/127304503-877f5290-e362-4f08-8ab9-51e3153adcea.png)

**Comment by greyli** [comment]
IMO, in this situation, you only need one schema with an optional `pagination` field.
